openadapt/strategies/mixins/summary.py

```diff
@@ -10,12 +10,13 @@ class MyReplayStrategy(SummaryReplayStrategyMixin):
 from openadapt.models import Recording
 from openadapt.strategies.base import BaseReplayStrategy
 
-
 from sumy.summarizers.lsa import LsaSummarizer
 from sumy.parsers.plaintext import PlaintextParser
 from sumy.nlp.tokenizers import Tokenizer
 from sumy.nlp.stemmers import Stemmer
 from sumy.utils import get_stop_words
+import nltk
```

Comment:
```
from loguru import logger
from sumy.nlp.stemmers import Stemmer
from sumy.nlp.tokenizers import Tokenizer
from sumy.parsers.plaintext import PlaintextParser
from sumy.summarizers.lsa import LsaSummarizer
from sumy.utils import get_stop_words
import nltk

from openadapt.models import Recording
from openadapt.strategies.base import BaseReplayStrategy
```

- Standard Imports followed by 3rd Party Imports followed by Local imports.
1 newline separation will be great between them. :)

- When it comes to ordering import ABC vs from ABC import XYZ. It is more readable to follow one specific format for all scripts/project. 

- When it comes to OpenAdapt. I have observed that we are following to  put from *** before import ***.
   See: https://github.com/MLDSAI/OpenAdapt/blob/main/openadapt/record.py#L18
  eg.
  ```
  from ABC import XYZ
  import ABC
  ```

- When it comes to order two similar imports  we are following alphabetical order: eg.
```
from A import W
from A import X
```

Reply:
Good point!! Addressed in [f2671f7](https://github.com/MLDSAI/OpenAdapt/pull/306/commits/f2671f7ba6715e5874fbec986ba9d01d62ba4d0e)